salt/core/ntp.sls

```diff
@@ -1,4 +1,24 @@
-# Disable pre-installed systemd-timesyncd as we prefer NTP.
+{% if grains['osrelease']|float >= 20.04 %}
+# systemd-timesyncd should be installed by default unless the NTP set up code below has ran against server.
+systemd-timesyncd:
+  pkg.installed:
+    - name: systemd-timesyncd
+  service.running:
+    - enable: True
+
+/etc/systemd/timesyncd.conf:
+  file.replace:
+    - pattern: "^#?NTP=.*"
+    - repl: 'NTP=0.uk.pool.ntp.org 1.uk.pool.ntp.org 2.uk.pool.ntp.org 3.uk.pool.ntp.org/'
```

Comment:
Why is there a trailing slash?

Reply:
That is a typo from the sed command we normally run. Fixed.